Mathlib/GroupTheory/GroupAction/Group.lean

```diff
@@ -4,6 +4,8 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Chris Hughes
 -/
 import Mathlib.Algebra.Group.Aut
+import Mathlib.Algebra.Invertible.Defs
+import Mathlib.Algebra.Invertible.Basic
```

Comment:
You shouldn't need to import both, since `Basic` imports `Defs`.
Also, how many files do these two `import`s add to the closure? If many, then it may be better to create a new file (e.g., `Mathlib.Algebra.Invertible.MulAction`).

Reply:
Otherwise LGTM. Thanks!
bors d+

Reply:
If I'm counting this correctly, `Mathlib/GroupTheory/GroupAction/Group.lean` on `master` currently has `224` dependencies in Mathlib, and `1038` dependencies if you also include packages and the Lean toolchain.

In contrast, `Mathlib/GroupTheory/GroupAction/Group.lean` on this branch has `232` dependencies in Mathlib, and `1046` dependencies if you also include packages and the Lean toolchain.

This doesn't seem like many to me, so I'll proceed to merge this PR in its current form.